plugins/CoreVue/Commands/Build.php

```diff
@@ -188,4 +192,25 @@ private function clearWebpackCache()
         $path = PIWIK_INCLUDE_PATH . '/node_modules/.cache';
         Filesystem::unlinkRecursive($path, true);
     }
+
+    private function checkNodeJsVersion(OutputInterface $output)
+    {
+        $nodeVersion = ltrim(`node -v`, 'v');
+        $npmVersion = ltrim(`npm -v`, 'v');
```

Comment:
guess a "normal" trim would also remove possible line endings, which might be better. Besides that this should be good to merge

Reply:
:+1: makes sense. I'll add a trim().